charts/consul/test/acceptance/tests/fixtures/cases/static-client-partition/patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: static-client
+spec:
+  template:
+    metadata:
+      annotations:
+        "consul.hashicorp.com/connect-inject": "true"
+        "consul.hashicorp.com/connect-service-upstreams": "static-server.default.secondary:8080"
```

Comment:
Because the commented out assertion is doing `curl localhost:1234`. Or we can change the way around. 
```suggestion
        "consul.hashicorp.com/connect-service-upstreams": "static-server.default.secondary:1234"
```

Reply:
i confused concepts :sigh:
Thank you!! 🙏